.github/workflows/test.yaml

```
@@ -1,4 +1,4 @@
-name: Test Samples (k3d and EKS)
+name: Test Samples
```

Maybe still keep k3d around for info on the cluster type being used?

```suggestion
name: Test Samples on k3d cluster
```

---

fixed